rocAL/include/pipeline/commons.h

```diff
@@ -211,11 +220,12 @@ class BatchRNG {
 /*! \brief MissingComponentsBehaviour for Webdataset
  *
  */
-enum MissingComponentsBehaviour {
+enum class MissingComponentsBehaviour {
```

Comment:
Non-blocking: Since this is an "enum class", consider explicitly specifying the underlying type (e.g., "enum class <Name> : int") to keep ABI stable and avoid surprises where different compilers choose different underlying types.